src/com/epam/Member.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.training;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+public class Member {
+    public Scanner input = new Scanner(System.in);
+    public Scanner input1 = new Scanner(System.in);
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
+    LocalDateTime now = LocalDateTime.now();
```

Comment:
it is not good to have now as field of the class, as you initialize it each time when sending the message